frontend/public/components/masthead.tsx

```diff
@@ -166,9 +166,9 @@ const Masthead_ = ({ flags }) => <header role="banner" className="navbar navbar-
   </div>
   <div className="nav navbar-nav navbar-right navbar-iconic navbar-utility">
     <div className="co-masthead__dropdowns">
-      {flags[FLAGS.CLUSTER_UPDATES_AVAILABLE] && <div className="co-masthead__updates">
+      {flags.CLUSTER_UPDATES_AVAILABLE && <div className="co-masthead__updates">
```

Comment:
Roll back the change to this line. The API is confusing, but what was there previously was correct.